src/views/actors/actor.json.ecr

```diff
@@ -50,6 +50,9 @@
     "url":<%= image.inspect %>
   },
   <%- end -%>
+  <%- if (attachments = actor.attachments) -%>
```

Comment:
this is fine, but at the top of this file you need to extend the context:
```
-    "https://w3id.org/security/v1"
+    "https://w3id.org/security/v1",
+    {
+      "schema":"http://schema.org#",
+      "PropertyValue":"schema:PropertyValue",
+      "value":"schema:value"
+    }
```
this will ensure that any json-ld clients (ktistec or others) will know how to expand and handle PropertyValue and value.